c2cgeoportal/scaffolds/create/vars_+package+.yaml_tmpl

```diff
@@ -115,6 +115,10 @@ vars:
             The GeoMapfish team
         smtp_server: smtp.example.com
 
+    headers_settings:
```

Comment:
I did not mean to change this to `headers_settings`. Just the variable in the Python code.